packages/terra-dropdown-button/src/terra-dev-site/doc/DropdownButton/DropdownButton.1.doc.mdx

```diff
@@ -37,6 +37,12 @@ This component requires the following peer dependencies be installed in your app
 
 <!-- AUTO-GENERATED-CONTENT:END -->
 
+## Usage
+
+```jsx
+import DropdownButton, { Item } from 'terra-dropdown-button';
+```
+
 ## Component Features
 
  * [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
```

Comment:
`
 * [Responsive Support](hhttps://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#responsive-support)
`
Extra `h` breaking this link here